airbyte-integrations/connectors/destination-s3/src/main/java/io/airbyte/integrations/destination/s3/avro/JsonSchemaType.java

```diff
@@ -4,39 +4,71 @@
 
 package io.airbyte.integrations.destination.s3.avro;
 
+import java.util.Arrays;
+import java.util.List;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 import org.apache.avro.Schema;
 
 /**
  * Mapping of JsonSchema types to Avro types.
  */
 public enum JsonSchemaType {
 
-  STRING("string", true, Schema.Type.STRING),
-  NUMBER("number", true, Schema.Type.DOUBLE),
-  INTEGER("integer", true, Schema.Type.INT),
-  BOOLEAN("boolean", true, Schema.Type.BOOLEAN),
-  NULL("null", true, Schema.Type.NULL),
-  OBJECT("object", false, Schema.Type.RECORD),
-  ARRAY("array", false, Schema.Type.ARRAY),
-  COMBINED("combined", false, Schema.Type.UNION);
+  STRING("string", true, null, Schema.Type.STRING),
+  NUMBER_INT("number", true, "integer", Schema.Type.INT),
+  NUMBER_LONG("number", true, "big_integer", Schema.Type.LONG),
```

Comment:
this doesn't fully align with https://docs.airbyte.com/understanding-airbyte/supported-data-types#the-types, is that intentional? I.e. `airbyte_type: big_integer` => `long`, rather than an arbitrary-precision avro datatype